src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -28,65 +28,72 @@ tribe.tickets.commerce.gateway.stripe = tribe.tickets.commerce.gateway.stripe ||
  */
 tribe.tickets.commerce.gateway.stripe.checkout = {};
 
-( ( $, obj, Stripe, ky ) => {
+(( $, tc, Stripe, ky ) => {
 	'use strict';
+
+	/**
+	 * The document element
+	 *
+	 * @since TBD
+	 *
+	 * @type {jQuery|HTMLElement}
+	 */
 	const $document = $( document );
 
 	/**
-	 * Pull the variables from the PHP backend.
+	 * The gateway.stripe object from the global tribe object
 	 *
 	 * @since TBD
 	 *
 	 * @type {Object}
 	 */
-	obj.checkout = tecTicketsCommerceGatewayStripeCheckout;
+	const obj = tc.gateway.stripe;
 
 	/**
-	 * Checkout Selectors.
+	 * The billing object from the global tribe object
 	 *
 	 * @since TBD
 	 *
 	 * @type {Object}
 	 */
-	obj.selectors = {
-		cardElement: '#tec-tc-gateway-stripe-card-element',
-		cardErrors: '#tec-tc-gateway-stripe-card-errors',
-		submitButton: '#tec-tc-gateway-stripe-checkout-button',
-	};
+	const billing = tc.billing;
 
 	/**
-	 * Stripe JS library.
+	 * Pull the variables from the PHP backend.
 	 *
 	 * @since TBD
 	 *
-	 * @type {Object|null}
+	 * @type {Object}
 	 */
-	obj.stripeLib = Stripe( obj.checkout.publishableKey );
+	obj.checkout = tecTicketsCommerceGatewayStripeCheckout;
 
 	/**
-	 * Stripe JS library elements.
+	 * Checkout Selectors.
 	 *
 	 * @since TBD
 	 *
-	 * @type {Object|null}
+	 * @type {Object}
 	 */
+	obj.selectors = {
+		cardNumber: '#tec-tc-gateway-stripe-card-number',
+		cardExpiry: '#tec-tc-gateway-stripe-card-expiry',
+		cardCvc: '#tec-tc-gateway-stripe-card-cvc',
+		cardZipWrapper: '#tec-tc-gateway-stripe-card-zip',
+		cardElement: '#tec-tc-gateway-stripe-card-element',
+		cardErrors: '#tec-tc-gateway-stripe-errors',
+		paymentElement: '#tec-tc-gateway-stripe-payment-element',
+		paymentMessage: '#tec-tc-gateway-stripe-payment-message',
+		submitButton: '#tec-tc-gateway-stripe-checkout-button'
```

Comment:
Same comment as above, if we have control over these (on the HTML) we should rely on classes rather than IDs